src/app/components/Citations/APACitation.jsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+
+class APACitation extends React.Component {
+  static getInitials(name) {
+    return name.trim().split(/\s+/).map(i => `${i.substr(0, 1).toUpperCase()}.`).join(' ');
+  }
+
+  componentWillMount() {
```

Comment:
does `componentDidMount` not receive props?

Reply:
For some reason with `componentDidMount` everything was rendered as `undefined` and then the parsing function was run. But it didn't update the displayed component, maybe because I'm not assigning anything to `state` in the function?

Reply:
Yeah I have thoughts on why this is, but I don't think doing it this way is going to cause problems and refactoring it so that `componentDidMount` will work is going to be another chunk of work so I think it's ok to leave it the way it is.